app/oauth2-server-config/client/admin/views/oauthApp.html

```diff
@@ -22,7 +22,7 @@
 						<div class="input-line double-col">
 							<label>{{_ "Redirect_URI"}}</label>
 							<div>
-								<input type="text" class="rc-input__element" name="redirectUri" value="{{data.redirectUri}}" />
+								<textarea class="rc-input__element" name="redirectUri" rows="4" style="height: auto">{{data.redirectUri}}</textarea>
 								<div class="settings-description secondary-font-color">{{_ "After_OAuth2_authentication_users_will_be_redirected_to_this_URL"}}</div>
```

Comment:
Also, can you please update the hint/description to describe you can have multiple URIs separated by a new line?